### PR TITLE
account to account tweaks

### PR DESCRIPTION
**Pull request by chrisnojima**
Account to account transaction details:
- [x] if sending wallet to wallet show the wallet name instead of your username

Transaction row (also on top of details)
- [x] if sending wallet to wallet w/o a note, use the small row type

Also moved the common props to reduce all this copy /paste in the stories

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/wallets/transaction-details/index.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ export type NotLoadingProps = {|
   transactionID?: string,
   you: string,
   yourRole: Types.Role,
+  // if 'you' is a wallet
+  youWereWallet: string,
```
Comment (buoyad) [inline shared/wallets/transaction-details/index.js]: confusing comment+naming here, took me a bit to see 'you' didn't mean `props.you`. Maybe `yourAccountName`?

File: shared/wallets/transaction-details/container.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     title: 'Transaction details',
     transactionID: tx.txID,
     you: stateProps.you,
+    youWereWallet: tx.sourceType ? tx.source : '',
```
Comment (buoyad) [inline shared/wallets/transaction-details/container.js]: `tx.sourceType === 'otherAccount'`

File: shared/wallets/transaction-details/index.js [hunk 3]
```diff
@@ -141,17 +143,27 @@ const propsToParties = (props: NotLoadingProps) => {
...
-      horizontal={true}
-      onClick={() => props.onShowProfile(props.you)}
-      underline={true}
-      username={props.you}
-      metaOne="You"
-      metaTwo={yourAccountID ? <SmallAccountID accountID={yourAccountID} /> : null}
-    />
-  )
+  const you =
+    props.counterpartyType === 'otherAccount' && props.youWereWallet ? (
```
Comment (buoyad) [inline shared/wallets/transaction-details/index.js]: `CounterParty` already has switching between showing `NameWithIcon` vs `CounterPartyText` etc. I think that should be making this decision too.

File: shared/wallets/transaction-details/container.js (hunk 2 above)
Comment (chrisnojima) [reply shared/wallets/transaction-details/container.js]: actually 'ownaccount'

Comment (buoyad) [reply shared/wallets/transaction-details/container.js]: whoops, sorry

Comment (buoyad) [reply shared/wallets/transaction-details/container.js]: Wait, we have `ownAccount` _and_ `otherAccount`? 🤔

File: shared/wallets/transaction-details/index.js (hunk 3 above)
Comment (buoyad) [reply shared/wallets/transaction-details/index.js]: `CounterPartyText` and `CounterPartyIcon` are used a lot, `CounterParty` is only in this file though

**Pull request merged**